### ISSUE TITLE
Farming/Gathering rebalance

### ISSUE THREAD

**Issue by BigWolfChris** [StrangeLoopGames/EcoSuggestions#1318]
As it stands, farming skills only use is for seed making.  The hoe calorie drop isn't a big deal tbf (and moot with a tractor) as you only plow once unless you make a mistake
So I suggest it gives some bonus to harvesting

But to prevent a gathering and farming combo upsetting the balance.  Perhaps gathering should only impact harvesting of wild plants, and farming level improves yield of farm plants

Gathering might want something added to keep it to make up for losing the ability to improve farm harvest?

**Comment by LadyNasse** [comment]
It would honestly be better to change the farming skill to affect things like plant health and yield, that kind of stuff, rather than changing gathering to be wild/domestic plant specific. A lot of skills have a synergy with others, i.e. mining and masonry or smelting, logging and carpentry. So keeping gathering and farming related to one another makes more sense, imo.

**Comment by Jacqhammer** [comment]
I agree with improving farming. I think farming should be set up more like engineering is, the farmer has more benches that do individual tasks. I also think that a farm plot bench would be a great idea. A craftable growing spot that is 5x5 and functions like a bench. The plot would have the same concepts as a work bench, you use seeds and fertilizer to grow crops over a given amount of time. You can have the seeds set dynamic to 25, 1 for each square, then use fertilizer as part of the recipe. 

As the game advances the farmer would get access to larger and more complex farm plots. Say 10x10 and 20x20 size benches that require items like pumps and pipes to move water. Adding to that idea, they could make green houses, same idea as a farm plot, but smaller. The main advantage of a green house would be its cuts time in half for growing crops, but produces a smaller yield do to the number of growing spots. So a 5x5 greenhouse would only grow 18 crops instead of 25, but in half the time. We get 18 growing spots from the idea that there are two layers of growing room in the greenhouse, 9 on the ground and 9 in the shelves. Just have the farm plot be made by the farming skill, have the gathering skill do the labor and fertilizers add in for growing crops. 

I'm aware that the idea of a farm plot bench goes against the idea of growing crops in the ground the way the game was designed. But let be honest, the game has had issues with that style of farming from day one. Problems like server lag delaying crop grow times, tree debris taking out crops, minor adjustments in biome creation killing crops over night. Based on the current system that is set up farming large areas of land is stupid hard. You could log on and find the best place to grow wheat is now some ones home. Imagine asking someone to rip up their home just because you want that land for wheat. There are too many minor factors that make farming the way the game is designed to unrealistic.

I know that idea also negates the steam tractor usage, but in reality the steam tractor is not used often. I say keep the steam tractor as it is just add attachments for mining, logging and road work to make the item more useable by the server. I think having a steam tractor attachment that can  do the work of a road tool would make everyone's lives better. 

I think based on technology we can either have small population servers with all these micro functions or have large population servers without all the micro functions. We can't have both. Its bad enough that server lag gets outrages when a population get to high. If they are going to get to the point of having multiple nations of people on a single server, a lot of the micro functions they have now will have to go.
